tests/phpunit/Admin/Admin_Page_Tests.php

```diff
@@ -145,15 +145,7 @@ public function test_filter_plugin_action_links() {
 		}
 		wp_set_current_user( $admin_user );
 		$action_links = $this->admin_page->filter_plugin_action_links( array(), $base_file, array(), 'all' );
-
-		$this->assertEquals(
-			sprintf(
-				'<a href="%1$s">%2$s</a>',
-				esc_url( admin_url( "tools.php?page=plugin-check&plugin={$base_file}" ) ),
-				esc_html__( 'Check this plugin', 'plugin-check' )
-			),
-			$action_links[0]
-		);
+		$this->assertEmpty( $action_links );
```

Comment:
@mukeshpanchal27 is this test correct? I looks like we assert the same thing twice but also not testing for when the link is added?

Can we add a test/assertion for when the Check this plugin link is added?

Reply:
I think we should add another test for this?

Reply:
Agreed, this is currently a duplicate. We should have one test here for a regular plugin and ensure that the link is added, and then _another_ test for the plugin checker plugin and ensure that the link is not added.

Reply:
Thank you for the feedback. In https://github.com/10up/plugin-check/pull/225/commits/f490742b350daad20de612a30c7440774de2657f i have added new unit test as requested.